web-analogs.md

```diff
@@ -0,0 +1,1035 @@
+---
+layout: code_comp_page
+title: HTML/CSS Analogs in Flutter
+sidebar: home_sidebar
+permalink: /web-analogs/
+---
+
+* TOC Placeholder
+{:toc}
+
+<div class="begin-examples"></div>
+This page is for users who are familiar with the HTML and CSS syntax for 
+arranging components of an application's UI. It maps HTML/CSS code snippets (on 
+the left) to their Flutter/Dart code equivalents (on the right).
```

Comment:
I'm viewing this on my laptop and there is only one column - the sections are on top of each other. Maybe delete the "on the left" and "on the right" text.